usaspending_api/core/validator/tinyshield.py

```diff
@@ -267,7 +307,23 @@ def apply_rule(self, rule):
                 child_rule['value'] = value
                 child_rule = self.promote_subrules(child_rule, v)
                 object_result[k] = self.apply_rule(child_rule)
-            return object_result
+            _return = object_result
+        # Any is a "special" type since it is is really a collection of other rules.
+        elif rule['type'] == 'any':
+            for child_rule in rule['models']:
+                child_rule['value'] = rule['value']
+                try:
+                    # First successful rule wins.
+                    return self.apply_rule(child_rule)
```

Comment:
Great job removing the multiple return statements from this function. Any desire to set `_return` and break the loop here instead of returning?

Reply:
The exception occurs if it runs through the loop and doesn't find a match.  I'm not sure how to circumvent the return without introducing yet another variable tracking whether or not it found an answer.  Which is worse?  A spurious `return` or an unnecessary artifact?

Reply:
Leave as-is. a "unnecessary artifact" is worse